src/utils/tracked-controls.js

```diff
@@ -171,7 +171,11 @@ function findMatchingControllerWebXR (controllers, idPrefix, handedness, index,
           if (controllerMatch) { break; }
         }
       } else {
-        controllerMatch = profiles.length > 0 && profiles[0].startsWith(idPrefix);
+        if (idPrefix === 'oculus-touch') {
+          controllerMatch = profiles.length > 0 && (profiles[0].startsWith(idPrefix) || profiles[0].startsWith('meta-quest-touch'));
```

Comment:
profile ids should not be hard coded here. Something is not working higher in the call chain

Reply:
Please feel free to commandeer this and make changes or just reference the changes made here in #5139. Hard coding it definitely is not a good solution, but I wasn't sure what you would like to do here to handle the two prefix mapping to one control problem

Reply:
> profile ids should not be hard coded here. Something is not working higher in the call chain

Happy to test out any new changes you make!

Reply:
I made changes to https://github.com/aframevr/aframe/pull/5139 Can you give it another try? Thanks